roofit/roofitcore/inc/RooAbsArg.h

```diff
@@ -92,16 +92,14 @@ class RooAbsArg : public TNamed, public RooPrintable {
   /// Does value or shape of this arg depend on any other arg?
   virtual Bool_t isDerived() const {
     return kTRUE ;
-    //std::cout << IsA()->GetName() << "::isDerived(" << GetName() << ") = " << (_serverList.GetSize()>0 || _proxyList.GetSize()>0) << std::endl ;
-    //return (_serverList.GetSize()>0 || _proxyList.GetSize()>0)?kTRUE:kFALSE;
   }
   Bool_t isCloneOf(const RooAbsArg& other) const ;
+  /// Does this arg depend on the value of any of the values in serverList?
   Bool_t dependsOnValue(const RooAbsCollection& serverList, const RooAbsArg* ignoreArg=0) const {
-    // Does this arg depend on the value of any of of the values in serverList?
     return dependsOn(serverList,ignoreArg,kTRUE) ;
   }
+  /// Does this arg depend on the value of server?
```

Comment:
might just be me, but questions as docstrings are weird :smile:

Reply:
True, I just moved the old-style in-function-body doc without updating it.